test/OpenSpaceToolkit/Astrodynamics/Trajectory/Propagator.test.cpp

```diff
@@ -2033,8 +2034,30 @@ TEST_F(OpenSpaceToolkit_Astrodynamics_Trajectory_Orbit_Models_Propagator, FromEn
         // and 1 default dynamics of integrating the Position Derivative
         EXPECT_TRUE(propagator.getDynamics().getSize() == 4);
```

Comment:
This one is still correct because the `Environment::Default` does not feature an `Earth` with an atm model, confirmed?

Reply:
Yup that's exactly it.

Reply:
Yup that's exactly it.

Reply:
Thanks for confirming 🙏